api/db-cdr/generate-cdr/init-new-cdr-db.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This generates new mysql database for a cdr version
+# note dev-up must be run to generate the schema
+# note run-local-data-migrations must be run to generate hard coded data from liquibase
+
+set -xeuo pipefail
+IFS=$'\n\t'
+
+# Todo change this script to use liquibase to make the db
+
+# INIT defaults FOR OPTIONAL PARAMS
+COPY_CDR_VERSION="";
+USAGE="./init-new-cdr-db.sh --cdr-version <VERSION> [--copy-cdr-version <VERSION>]"
+while [ $# -gt 0 ]; do
+  echo "1 is $1"
+  case "$1" in
+    --cdr-version) CDR_VERSION=$2; shift 2;;
+    --copy-cdr-version) COPY_CDR_VERSION=$2; shift 2;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+# Todo check versions are of form YYYYMMDD
+if [ -z "${CDR_VERSION}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+
+# variables
+tmp_dir="/tmp"
+cdr_schema_file="cdr_schema.sql"
+public_schema_file="public_schema.sql"
+hard_data_file="hard_data.sql"
+new_cdr_db_name="cdr$CDR_VERSION"
+copy_cdr_db_name="cdr$COPY_CDR_VERSION"
+
+echo "Making a copy of $copy_cdr_db_name to $new_cdr_db_name"
+
+# create new cdr database
+mysql -h ${DB_HOST} --port ${DB_PORT} -u root -p${MYSQL_ROOT_PASSWORD} -e "drop database if exists $new_cdr_db_name"
+mysql -h ${DB_HOST} --port ${DB_PORT} -u root -p${MYSQL_ROOT_PASSWORD} -e "create database $new_cdr_db_name"
+
+# Dump copy cdr schema and import to new database
```

Comment:
I thought we were going to use Liquibase directly for generating the schema in the new database?

(If you were planning on doing that in a follow-up, you could probably get rid of this file?)

Reply:
right. i do. i removed this.